### PR TITLE
fixed overlap header button colors

### PR DESCRIPTION
**Pull request by codecalm**
**Before:**

Screenshot 2022-01-21 at 21 36 21

**After:**

Screenshot 2022-01-21 at 21 35 34

### CODE REVIEW

File: _config.yml [hunk 1]
```diff
@@ -68,10 +68,10 @@ defaults:
...
     values:
       layout: default
-  - scope:
-      type: "docs"
-    values:
-      layout: docs
+#  - scope:
+#      type: "docs"
+#    values:
+#      layout: docs
```
Comment (rjd22) [inline _config.yml:74]: Is this commented out for testing or something else?

**Review by rjd22 — APPROVED** [review]
Looks good to me only 1 small question

File: _config.yml (hunk 1 above)
Comment (codecalm) [reply _config.yml]: damn, I forgot to uncomment this 😂  fixed

**Review by rjd22 — APPROVED** [review]

**Comment by rjd22** [comment]
Looks way better this way.

**Comment by rjd22** [comment]
A lot of few changed by the merge. I'm not sure this was intentional or not.

**Comment by codecalm** [comment]
fixed

**Pull request merged**